assets/js/components/HostDetails/HostDetails.jsx

```diff
@@ -140,18 +151,63 @@ function HostDetails({
         {versionWarningMessage && (
           <WarningBanner>{versionWarningMessage}</WarningBanner>
         )}
-        <div className="mt-4 bg-white shadow rounded-lg py-4 px-8">
-          <ListView
-            orientation="vertical"
-            data={[
-              { title: 'Name', content: hostname },
-              {
-                title: 'Cluster',
-                content: <ClusterLink cluster={cluster} />,
-              },
-              { title: 'Agent version', content: agentVersion },
-            ]}
-          />
+        <div className="flex xl:flex-row flex-col">
+          <div className="mt-4 bg-white shadow rounded-lg py-4 px-8 xl:w-2/5 mr-4">
+            <ListView
+              className="grid-rows-3"
+              orientation="vertical"
+              data={[
+                {
+                  title: 'Cluster',
+                  content: <ClusterLink cluster={cluster} />,
+                },
+                { title: 'Agent Version', content: agentVersion },
+                { title: 'IP addresses', content: ipAddresses.join(',') },
+              ]}
+            />
+          </div>
+          <div className="flex flex-col mt-4 bg-white shadow rounded-lg pt-8 px-8 xl:w-2/5 mr-4">
+            <div className="flex justify-between mb-2">
+              <h1 className="text-2xl font-bold">Saptune Summary</h1>
+              <Button
+                type="primary-white-fit"
+                className="border-green-500 border"
+                size="small"
+                disabled={!isVersionSupported(saptuneVersion)}
+              >
+                View Details
+              </Button>
+            </div>
+            <ListView
+              className="grid-rows-2"
+              orientation="vertical"
+              data={[
+                {
+                  title: 'Package',
+                  content: <SaptuneVersion version={saptuneVersion} />,
+                },
+                {
+                  title: 'Tunning',
+                  content: <SaptuneTuningState state={saptuneTuning} />,
+                },
+                {
+                  title: 'Configured Version',
+                  content: saptuneConfigureVersion || '-',
+                },
+              ]}
+            />
+          </div>
+          <div className="mt-4 bg-white shadow rounded-lg py-4 xl:w-1/4">
```

Comment:
@dottorblaster @nelsonkopliku I guess we will need to have some `checksAvailable` kind of variable to decide if we put this `coming soon` or the results box.

Reply:
I guess, so. We'll be working on that soon.